pycodestyle.cfg

```diff
@@ -0,0 +1,10 @@
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
```

Comment:
I know we "borrowed" from eva, but we can just remove the copyright stuff from these 2 files